CLIENT-SPECIFICATION.md

```diff
@@ -85,7 +85,9 @@ The main version of every page is stored inside (but not directly) the `pages` d
    - `osx/`
    - ...etc.
 
-Additional platforms MAY be added in the future. Clients MAY NOT support new platforms (though such support is RECOMMENDED), but MUST NOT break if additional platforms are added.
+It is RECOMMENDED that client supports `macos` as an alias for `osx`.
+
+Additional platforms MAY be added in the future. Clients MAY NOT support new platforms (though such support is RECOMMENDED), but MUST NOT break if additional platforms are added. 
```

Comment:
This is unrelated, but I noticed that the language here is wrong. Instead of `Clients MAY NOT support new platforms` it should be `Clients MAY not support new platforms`. Otherwise it would mean that supporting new platforms is forbidden.

Maybe it could be fixed in this PR as well (I thought about opening a separate PR, but that would conflict with yours).

Reply:
I think you want it changed to `SHOULD NOT`? Not sure why lowercasing the `not` here would change the meaning of the sentence.

However, this whole sentence can probably be reworked as I'm not sure why there's also the "though such support is RECOMMENDED" when the behavior is not recommended? What's the sentiment that's being expressed in this sentence?

Reply:
I think it's the other way around? The point is that tldr may add new platform directories (e.g. `android`), and it's possible that a certain client doesn't support it (yet, it's still compliant).

If we use `MAY NOT` or `SHOULD NOT`, then the client is forbidden from adding support for this platform, right? That doesn't make sense at all.

Reply:
So should the wording here be `Clients MAY support new platforms (though such support is RECOMMENDED), but MUST NOT break if additional platforms are added to tldr-pages` (drop the `not` altogether, not just lowercase it)?

Reply:
What about `While clients may not support new platforms automatically (though such support is RECOMMENDED), they MUST NOT break if additional platforms are added to tldr-pages.`?

Reply:
Emphasis added on a proposed tweak:
> While clients __do not need to__ support new platforms automatically (though such support is RECOMMENDED), they MUST NOT break if additional platforms are added to tldr-pages.

using `may not support` implies to me that they should not have this support at all and contradicts the recommendation. My tweak here I think implies what we want?

Reply:
Sounds good, @MasterOdin.

Reply:
Done